mitodl/ocw-studio
# Trigger concourse build via API

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #784 

#### What's this PR do?
Fixes some issues with the build status:
 - Trigger builds explicitly using the concourse REST API. We are saving the build id in the database so that we can look up the build status directly
 - Update status code to retrieve status based on the latest build id

#### How should this be manually tested?
If you don't have concourse environment variables set up already, the most straightforward way is to get them from RC. Then trigger a draft build. It should start off as "Not started", then "In progress" for a while, then "Success" or "Failed". If it succeeded, you should be able to view the published website using the link in the publish drawer
